### ISSUE TITLE
output error of the normal  about pytorch and DALI  

### ISSUE THREAD

**Issue by swpucl** [tanglang96/DataLoaders_DALI#22]
pytorch:  transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
output:tensor([[[[ 0.3309,  0.3481,  0.3652,  ...,  0.9817,  0.9817,  0.9817],
          [ 0.3481,  0.3652,  0.3823,  ...,  0.9988,  0.9988,  0.9988],
          [ 0.3652,  0.3823,  0.3823,  ...,  1.0159,  1.0159,  1.0159],
          ...,
          [-1.5014, -1.5185, -1.5528,  ...,  0.3994,  0.3481,  0.3481],
          [-1.5014, -1.5185, -1.5528,  ...,  0.3994,  0.3481,  0.3823],
          [-1.5014, -1.5185, -1.5528,  ...,  0.3994,  0.3994,  0.3823]],

         [[ 0.6429,  0.6954,  0.7129,  ...,  1.4132,  1.4132,  1.4132],
          [ 0.6604,  0.7129,  0.7304,  ...,  1.4307,  1.4307,  1.4307],
          [ 0.6779,  0.7304,  0.7304,  ...,  1.4482,  1.4482,  1.4482],
          ...,
          [-1.0903, -1.0903, -1.0553,  ...,  0.8529,  0.8004,  0.8004],
          [-1.0903, -1.0903, -1.0553,  ...,  0.8529,  0.8004,  0.8354],
          [-1.0903, -1.0903, -1.0553,  ...,  0.8529,  0.8529,  0.8354]],

         [[ 0.8797,  0.9145,  0.9319,  ...,  1.6291,  1.6291,  1.6291],
          [ 0.8971,  0.9319,  0.9494,  ...,  1.6465,  1.6465,  1.6465],
          [ 0.9145,  0.9494,  0.9494,  ...,  1.6640,  1.6640,  1.6640],
          ...,
          [-0.6193, -0.6193, -0.6193,  ...,  1.2457,  1.1934,  1.2282],
          [-0.6193, -0.6193, -0.6193,  ...,  1.2457,  1.1934,  1.2631],
          [-0.6193, -0.6193, -0.6193,  ...,  1.2457,  1.2457,  1.2631]]]])

DALI:ops.CropMirrorNormalize(device = "gpu",
                                            output_dtype = types.FLOAT,
                                            image_type = types.RGB,
                                            output_layout=types.NCHW,
                                            mean = [0.485*255, 0.456*255, 0.406*255],
                                            std = [0.229*255, 0.224*255, 0.225*255]
                                            )
output:tensor([[[[ 0.3138,  0.3309,  0.3481,  ...,  0.9646,  0.9646,  0.9646],
          [ 0.3309,  0.3481,  0.3652,  ...,  0.9817,  0.9817,  0.9817],
          [ 0.3481,  0.3652,  0.3652,  ...,  0.9988,  0.9988,  0.9988],
          ...,
          [-1.5185, -1.5357, -1.5699,  ...,  0.3823,  0.3309,  0.3309],
          [-1.5185, -1.5357, -1.5699,  ...,  0.3823,  0.3481,  0.3652],
          [-1.5185, -1.5357, -1.5699,  ...,  0.3823,  0.3823,  0.3652]],

         [[ 0.6254,  0.6779,  0.6954,  ...,  1.3957,  1.3957,  1.3957],
          [ 0.6429,  0.6954,  0.7129,  ...,  1.4132,  1.4132,  1.4132],
          [ 0.6604,  0.7129,  0.7129,  ...,  1.4307,  1.4307,  1.4307],
          ...,
          [-1.1078, -1.0903, -1.0553,  ...,  0.8704,  0.8004,  0.8004],
          [-1.1078, -1.0903, -1.0553,  ...,  0.8704,  0.8179,  0.8354],
          [-1.1078, -1.0903, -1.0553,  ...,  0.8704,  0.8529,  0.8354]],

         [[ 0.8622,  0.8971,  0.9145,  ...,  1.6291,  1.6291,  1.6291],
          [ 0.8797,  0.9145,  0.9319,  ...,  1.6465,  1.6465,  1.6465],
          [ 0.8971,  0.9319,  0.9319,  ...,  1.6640,  1.6640,  1.6640],
          ...,
          [-0.6367, -0.6367, -0.6193,  ...,  1.2631,  1.1934,  1.2108],
          [-0.6367, -0.6367, -0.6193,  ...,  1.2631,  1.2108,  1.2457],
          [-0.6367, -0.6367, -0.6193,  ...,  1.2631,  1.2457,  1.2457]]]],
       device='cuda:3')
